test/vstest.console.UnitTests/vstest.console.UnitTests.csproj

```diff
@@ -1,30 +1,33 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
-  <PropertyGroup>
-    <TestPlatformRoot Condition="$(TestPlatformRoot) == ''">..\..\</TestPlatformRoot>
-    <TestProject>true</TestProject>
-  </PropertyGroup>
-  <Import Project="$(TestPlatformRoot)scripts/build/TestPlatform.Settings.targets" />
-  <PropertyGroup>
-    <OutputType Condition=" '$(TargetFramework)' != 'net451' ">Exe</OutputType>
-    <TargetFrameworks>netcoreapp2.1;net451</TargetFrameworks>
-    <AssemblyName>vstest.console.UnitTests</AssemblyName>
-  </PropertyGroup>
-  <ItemGroup>
-    <ProjectReference Include="..\..\src\vstest.console\vstest.console.csproj" />
-    <ProjectReference Include="..\..\src\Microsoft.TestPlatform.Utilities\Microsoft.TestPlatform.Utilities.csproj">
-      <FromP2P>true</FromP2P>
-    </ProjectReference>
-    <ProjectReference Include="..\..\src\Microsoft.TestPlatform.PlatformAbstractions\Microsoft.TestPlatform.PlatformAbstractions.csproj">
-      <FromP2P>true</FromP2P>
-    </ProjectReference>
-  </ItemGroup>
-  <ItemGroup Condition=" '$(TargetFramework)' == 'net451' ">
-    <Reference Include="System.Runtime" />
-    <Reference Include="System" />
-    <Reference Include="Microsoft.CSharp" />
-    <Reference Include="System.Xml" />
-    <Reference Include=" System.Runtime.Serialization" />
-  </ItemGroup>
-  <Import Project="$(TestPlatformRoot)scripts\build\TestPlatform.targets" />
+    <PropertyGroup>
+        <TestPlatformRoot Condition="$(TestPlatformRoot) == ''">..\..\</TestPlatformRoot>
+        <TestProject>true</TestProject>
+    </PropertyGroup>
+    <Import Project="$(TestPlatformRoot)scripts/build/TestPlatform.Settings.targets" />
+    <PropertyGroup>
+        <OutputType Condition=" '$(TargetFramework)' != 'net451' ">Exe</OutputType>
+        <TargetFrameworks>netcoreapp2.1;net451</TargetFrameworks>
+        <AssemblyName>vstest.console.UnitTests</AssemblyName>
+    </PropertyGroup>
+    <ItemGroup>
+        <ProjectReference Include="..\..\src\vstest.console\vstest.console.csproj" />
+        <ProjectReference Include="..\..\src\Microsoft.TestPlatform.Utilities\Microsoft.TestPlatform.Utilities.csproj">
+            <FromP2P>true</FromP2P>
+        </ProjectReference>
+        <ProjectReference Include="..\..\src\Microsoft.TestPlatform.PlatformAbstractions\Microsoft.TestPlatform.PlatformAbstractions.csproj">
+            <FromP2P>true</FromP2P>
+        </ProjectReference>
+    </ItemGroup>
+    <ItemGroup>
+        <PackageReference Include="System.ValueTuple" Version="4.5.0" />
```

Comment:
What is the change in this whole file? Formatting and the value tuple reference? Could we do without it? Where is it used? I don't see any mention of value tuple apart from here.

Reply:
Formatting is probably updated by VS, I haven't directly edited this file. ValueTuple is used in the tests I wrote. In `/test/vstest.console.UnitTests/Processors/EnvironmentArgumentProcessorTests.cs` specifically.  

We can avoid using it, but code is a bit cleaner with it.

Reply:
Ah this is just unittests, nevermind it then.